docker-compose-perf.yml

```diff
@@ -564,6 +596,7 @@ services:
       - agreement
       - all-services
       - transfers-test
+      - quotes-test
```

Comment:
Why do we need kafka for quotes?

Reply:
From my understanding

The central ledger we use for onboarding produces a notification when adding initial limits and positions.
Without it, it will error out. The quoting service does a validation for the participant currency accounts which won't be created without kafka.